packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -921,7 +921,7 @@ export function useGlobalStylesOutput() {
 				isGlobalStyles: true,
 			},
 			{
-				css: customCSS,
+				css: customCSS ?? '',
```

Comment:
Should fix failing e2e tests. It probably won't be needed once `customCSS` preset is part of the editor settings on the server side.